clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -453,6 +468,12 @@ public synchronized void endBatchWrite() {
      * the last SST file written is finished.
      */
     rocksDBSstFileWriter.ingestSSTFiles(rocksDB, columnFamilyHandleList);
+    createSnapshot();
+  }
+
+  @Override
+  public synchronized void createSnapshot() {
+    rocksDBSstFileWriter.createSnapshot(rocksDB);
```

Comment:
We should addd a check here too. If the feature is not enabled, calling this method would result in one exception.

Reply:
thanks for the catch